tests/cli/test_main_remove.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+from os.path import (
+    exists,
+    join,
+)
+
+import pytest
+
+from conda.exceptions import (
+    PackagesNotFoundError,
+)
+from conda.gateways.disk.delete import path_is_clean
+from conda.testing import CondaCLIFixture, TmpEnvFixture
+from conda.testing.integration import (
+    PYTHON_BINARY,
+    package_is_installed,
+)
+
+
+def test_remove_all(tmp_env: TmpEnvFixture, conda_cli: CondaCLIFixture):
+    with tmp_env("python") as prefix:
+        assert exists(join(prefix, PYTHON_BINARY))
+        assert package_is_installed(prefix, "python")
+
+        # regression test for #2154
+        with pytest.raises(PackagesNotFoundError) as exc:
+            conda_cli("remove", f"--prefix={prefix}", "python", "foo", "numpy")
```

Comment:
`conda_cli` does not magically insert `--yes` like the older `run_command` implementation did so we need to be explicit

```suggestion
            conda_cli("remove", f"--prefix={prefix}", "python", "foo", "numpy", "--yes")
```